Admin/AbstractAdmin.php

```diff
@@ -2017,7 +2017,7 @@ public function buildBreadcrumbs($action, MenuItemInterface $menu = null)
             E_USER_DEPRECATED
         );
         if ($this->breadcrumbsBuilder === null) {
-            $this->breadcrumbsBuilder = new BreadcrumbsBuilder();
+            $this->breadcrumbsBuilder = new BreadcrumbsBuilder($this->getConfigurationPool()->getContainer()->getParameter('sonata.admin.configuration.breadcrumbs'));
```

Comment:
Can't we switch this? So the breadcrumb builder asks the admin for this property?